applications/StructuralMechanicsApplication/custom_python/add_custom_strategies_to_python.cpp

```diff
@@ -35,6 +35,7 @@
 #include "solving_strategies/convergencecriterias/convergence_criteria.h"
 #include "custom_strategies/custom_convergencecriterias/displacement_and_other_dof_criteria.h"
 #include "custom_strategies/custom_convergencecriterias/residual_displacement_and_other_dof_criteria.h"
+#include "custom_strategies/custom_convergencecriterias/error_mesh_criteria.h"
```

Comment:
ZZ estimator and remeshing should go in two different PRs (as i told you in words)

Reply:
Ok